src/components/HdTileSelect.vue

```diff
@@ -0,0 +1,114 @@
+<template>
+  <section class="field tile-select">
+    <section class="tile-select__items">
+      <hd-tile-select-item
+        class="tile-select__item"
+        v-for="item in itemsMapped"
+        :key="item.value"
+        :value="item.value"
+        :text="item.text"
+        :model="value"
+        @input="$emit('input', $event)"
+      />
+      <hd-tile-select-editable-item
+        v-if="acceptNewValue"
+        class="tile-select__item"
+        v-bind="$attrs"
+        v-model="customInputModel"
+        :text="customInputText"
+        :model="value"
+      />
+    </section>
+  </section>
+</template>
+
+<script>
+import _get from 'lodash/get';
+import HdTileSelectItem from 'homeday-blocks/src/components/HdTileSelectItem.vue';
+import HdTileSelectEditableItem from 'homeday-blocks/src/components/HdTileSelectEditableItem.vue';
+
+export default {
+  name: 'HdTileSelect',
+  inheritAttrs: false,
+  components: {
+    HdTileSelectItem,
+    HdTileSelectEditableItem,
+  },
+  data() {
+    return {
+      customInputModel: '',
+    };
+  },
+  props: {
+    /*
```

Comment:
If it's not used, I think we can remove the whole commented section.

Reply:
I didn't get your point here but it is just to document how `items` can be passed to this component (TS feelings 😅)

Reply:
I guess it kinda guides about this variable, and it's not a problem from my side, it's already will be deleted by webpack after rendering process.

Reply:
Ah ok. Haven't read the text and haven't seen, it's an explanation. Shame on me. 😁  Then it's fine. It just looked like commented code that isn't used and can be removed.

Reply:
I've added it as `JSDoc` here: https://github.com/homeday-de/homeday-blocks/pull/465/commits/5fb70fbbbf72bb6e79ff2a4060b6ed75128cf9f9